README.md

```diff
@@ -99,14 +100,15 @@ Note: *You should run this command as non-root*
 It's also fully supported to use `podman` as root, but some of the arguments
 here need to change for that.
 
-```
+```sh
 cosa() {
    env | grep COREOS_ASSEMBLER
+   local -r cosa_name="cosa-$(head /dev/urandom | tr -dc a-z | head -c 6)"
```

Comment:
I think it's simpler to tell people to just omit `--name` if they want this - podman takes a page from docker and already knows how to allocate a random name (in a cooler way IMO :wink: ).

Perhaps break out a quick separate section around "running multiple instances" maybe?

Reply:
Indeed! Fixed